torch/csrc/jit/ir.h

```diff
@@ -700,9 +700,15 @@ struct Block {
   Graph * owningGraph() {
```

Comment:
Is it possible to delete these now that this and `Node * owningNode()` are `const` versions?

Reply:
didn't check the code, but maybe the caller would want a mutable pointer to the owning node?